### ISSUE TITLE
Make the location where the Docker image copies JARs configurable

### ISSUE THREAD

**Issue by jsanda** [datastax/management-api-for-apache-cassandra#50]
In Dockerfile-3_11 and in Dockerfile-4_0 jars are copied as follows:

```
COPY --from=builder /build/management-api-common/target/datastax-mgmtapi-common-0.1.0-SNAPSHOT.jar /etc/cassandra/
COPY --from=builder /build/management-api-agent/target/datastax-mgmtapi-agent-0.1.0-SNAPSHOT.jar /etc/cassandra/
COPY --from=builder /build/management-api-server/target/datastax-mgmtapi-server-0.1.0-SNAPSHOT.jar /opt/mgmtapi/
COPY --from=builder /build/management-api-shim-3.x/target/datastax-mgmtapi-shim-3.x-0.1.0-SNAPSHOT.jar /opt/mgmtapi/
COPY --from=builder /build/management-api-shim-4.x/target/datastax-mgmtapi-shim-4.x-0.1.0-SNAPSHOT.jar /opt/mgmtapi/
```

I am working on Medusa integration with cass-operator for backup/restore support. Medusa needs access to `/etc/cassandra/cassandra.yaml`. I did some testing with creating a mount point at `/etc/cassandra`. The image copies the JARs into `/etc/cassandra`. The directory is then mounted over and the files are not accessible.

Can we change the location to /tmp and and have `docker-entrypoint.sh` copy them over to `/etc/cassandra`?

**Comment by bradfordcp** [comment]
I would be careful with /tmp as well. Would placing the files under `/opt/mgmtapi/`hurt anything then use the entrypoint to copy into /etc/cassandra be a problem?

**Comment by jsanda** [comment]
Yeah `/tmp` is risky. I don't think there would be any issue with putting them under `/opt/mgmtapi` but I need to go back and look and test to verify.

**Comment by emerkle826** [comment]
This is resolved as of [version 0.1.15](https://github.com/datastax/management-api-for-apache-cassandra/releases/tag/v0.1.15)
